ark/utils/load_utils.py

```diff
@@ -324,10 +294,6 @@ def load_imgs_from_dir(data_dir, files=None, match_substring=None, trim_suffix=N
     if channel_indices and multitiff:
         img_data = img_data[:, :, :, channel_indices]
 
-    # check to make sure that dtype wasn't too small for range of data
-    if np.min(img_data) < 0:
```

Comment:
We should keep this around, and add it to the other load functions, but make it a warning, not an error. In general it probably means something went wrong if you're loading images with negative values, so this would be a good thing to have a basic check. It's also possible that the test_img had a different dtype than the other images in the directory. 

There are some cases where the user might be loading z-scored images, where negatives are expected, so we don't automatically want to raise an error, but good to flag